strings/tips/xeno.txt

```diff
@@ -38,3 +38,28 @@ As a spitter, boiler, praetorian, or the Queen take advantage of unguarded barri
 As a boiler or the Queen, you have the most powerful corrosive acid for melting things.
 As the Shrike, take advantage of your psychic abilities. Use Psychic Fling and Unrelenting Force to displace a marine or even push certain and hazardous objects to your enemies and vice versa.
 As the Queen, your screech does not only affect the mobs on view, but affects through walls or gases. However, the effect of the screech will be reduced.
+As a drone, hivelord, hivemind, shrike, and queen, large maze is almost always a good investment.
+As a drone, hivelord, hivemind, shrike, and queen, clever with your walling. Try to allow for as much fluid movement as possible.
```

Comment:
There should be a linking verb here. "**_be_** clever with your walling." would suffice.